layers/meta-balena-beaglebone/recipes-bsp/u-boot/u-boot-ti-staging/0001-Integrate-with-Balena-u-boot-environment.patch

```diff
@@ -25,6 +25,29 @@ index 9d2a225e7e..0d2ae1a889 100644
  		"env exists devplist || setenv devplist 1; "              \
  		"for distro_bootpart in ${devplist}; do "                 \
  			"if fstype ${devtype} "                           \
--- 
-2.25.1
-
+Index: git/configs/j721e_beagleboneai64_a72.config
```

Comment:
Which config is the one actually used? It should only be one of j721e_beagleboneai64_a72.config or j721e_evm_a72_defconfig so let's only modify the one that it's actually included for the bb ai 64 board